ReactWindows/ReactNative/Modules/Storage/AsyncStorageModule.cs

```diff
@@ -1,15 +1,34 @@
-﻿using ReactNative.Bridge;
+﻿using Newtonsoft.Json.Linq;
+using ReactNative.Bridge;
+using System;
+using Windows.Storage;
 
 namespace ReactNative.Modules.Storage
 {
     /// <summary>
     /// The asynchronous storage module.
     /// </summary>
-    /// <remarks>
-    /// TODO: implement.
-    /// </remarks>
-    public class AsyncStorageModule : NativeModuleBase
+    public class AsyncStorageModule : ReactContextNativeModuleBase
```

Comment:
At least for now, do we really need this to be ReactContextNativeModuleBase?  I don't see any references to `this.Context` anywhere...